src/Raven.Studio/typescript/models/database/index/index.ts

```diff
@@ -100,6 +100,7 @@ class index {
                 this.isAutoIndex(true);
                 return "Auto Map-Reduce";
             default:
+                this.isAutoIndex(this.name.startsWith("Auto"));
```

Comment:
just in case I would suggest changing this to 'Auto/'

Reply:
done